src/interactive/Tooltip.jsx

```diff
@@ -167,9 +175,10 @@ Tooltip.propTypes = {
 	trigger: PropTypes.element.isRequired,
 	content: PropTypes.element,
 	align: PropTypes.oneOf(["left", "right", "center"]).isRequired,
+	popAbove: PropTypes.bool,
 	className: PropTypes.string,
 	isActive: PropTypes.bool,
-	manualToggle: PropTypes.func,
+	manualToggle: PropTypes.bool,
```

Comment:
This looks like this could be a breaking change. This component doesn't have much, if any usage right now, so it's probably ok as long as:
- `pro-web` isn't using this prop
- `mup-web` isn't using this prop
- other `manualToggle` props used in MWC accept `bool` (helps keep the interface predictable)

Reply:
`manualToggle` is accepts `func` for `Dropdown`, and nobody was using this prop. It was accidentally copied over from `Dropdown` proptypes

Reply:
would it be safe to change the `Dropdown` `manualToggle` propType to `bool` so that our prop signatures are consistent?